platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less

```
@@ -53,6 +53,16 @@
   }
 }
 
+#resetButtonFilterPeopl {
```

```suggestion
.peopleAdvancedFilterResetButton {
```
try to move it directly under #peopleListApplication

---

fixed !